packages/drivers/routerlicious-driver/src/test/errorUtils.spec.ts

```diff
@@ -229,8 +241,10 @@ describe("ErrorUtils", () => {
 				},
 				handler,
 			);
+
+			assert(isIThrottlingWarning(error));
 			assertExpectedMessage(error.message);
-			assert.strictEqual(error.errorType, DriverErrorType.throttlingError);
+			assert.strictEqual(error.errorType, RouterliciousErrorTypes.throttlingError);
```

Comment:
We can remove this line; `isIThrottlingWarning` already checks the value of `errorType`.